test/unit/core.js

```diff
@@ -1709,3 +1709,45 @@ QUnit.test( "Iterability of jQuery objects (gh-1693)", function( assert ) {
 		assert.ok( true, "The browser doesn't support Symbols" );
 	}
 } );
+
+QUnit[ jQuery.Deferred ? "test" : "skip" ]( "jQuery.readyException (original)", function( assert ) {
+	assert.expect( 1 );
+
+	var message,
+		origWindowSetTimeout = window.setTimeout;
+
+	window.setTimeout = function( fn ) {
```

Comment:
Yeah, I was thinking about sinon but it's not activated at all for this module, I'd need to enter the sinon sandbox to be able to use spies, no?